analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/ObjectsShouldNotBeDisposedMoreThanOnce.cs

```diff
@@ -151,4 +166,33 @@ public static void LoopWithBreak(System.Collections.Generic.IEnumerable<string>
             }
         }
     }
+
+    public static void Loop(System.Collections.Generic.IEnumerable<string> list, bool condition, IInterface1 instance1)
```

Comment:
What is the `1` for in `interface1`?

Reply:
No idea. Inherited from the previous tests.
It's renamed.

Reply:
There's still a lot of `instance1` arguments where there's not `instance2`. They could be just `d`, `arg` or `value` instead